src/main.rs

```diff
@@ -138,15 +137,35 @@ fn main() {
     };
     println!("reduce/reduce: {}\nshift/reduce: {}", stable.reduce_reduce, stable.shift_reduce);
 
-    // Sync up the IDs of terminals in the lexer and parser
-    let mut rule_ids = HashMap::<&str, u16>::new();
-    for term_idx in grm.iter_term_idxs() {
-        rule_ids.insert(grm.term_name(term_idx).unwrap(), u16::try_from(usize::from(term_idx)).unwrap());
+    {
```

Comment:
Ok, I see now my confusion was with the for look that was deleted, not the new code. Sigh.